.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: node_js
+
+node_js:
+  - "6"
+
+sudo: enabled
+
+before_install:
+  - pip install psutil
```

Comment:
Is this necessary ?
I don't remember that I did run this command before running ./launch script.

Reply:
launch script didn't install pip package properly on travis-ci.
So I install packcage manually.